<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabCooling">
       <attribute name="title">
        <string>Cooling</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabLighting">
       <attribute name="title">
        <string>Lighting</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QtChart.QChartView" name="frame">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>180</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>180</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPushButton" name="pushButtonLogo">
           <property name="geometry">
            <rect>
             <x>44</x>
             <y>73</y>
             <width>91</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>22</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>NZXT</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Select the LED you want to edit:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QCheckBox" name="checkBoxLogoLed">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>LOGO LED</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>FIXED - Default</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="checkBoxRingLed">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>RING LED</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>FIXED - Default</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButtonSave">
          <property name="text">
           <string>SAVE CHANGES</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_2_1">
           <attribute name="title">
            <string>Preset</string>
           </attribute>
           <widget class="QMdiArea" name="mdiArea">
            <property name="geometry">
             <rect>
              <x>206</x>
              <y>7</y>
              <width>591</width>
              <height>411</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
           <widget class="QWidget" name="verticalLayoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>181</width>
              <height>91</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayoutPresetLeds">
             <item>
              <widget class="QComboBox" name="comboBoxPresetModes"/>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Animation Speed:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSliderASpeed">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>4</number>
               </property>
               <property name="pageStep">
                <number>1</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_2_2">
           <attribute name="title">
            <string>Smart</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabOverclock">
       <attribute name="title">
        <string>Overclocking</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>858</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>&amp;Device</string>
    </property>
    <widget class="QMenu" name="menu_Select_Device">
     <property name="title">
      <string>&amp;Select Device:</string>
     </property>
     <addaction name="actiondummy"/>
    </widget>
    <addaction name="menu_Select_Device"/>
    <addaction name="actionReload"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDevice"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>&amp;Refresh Devices</string>
   </property>
  </action>
  <action name="actiondummy">
   <property name="text">
    <string>&amp;dummy</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtChart.QChartView</class>
   <extends>QFrame</extends>
   <header location="global">PyQt5</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
